### PR TITLE
 Make initialize() pass extra keyword arguments to LDAPObject 

### PR DESCRIPTION
**Pull request by encukou**
Fixes: #249 

Also, documentation for these should now match the code.

### CODE REVIEW

**Comment by encukou** [comment]
@tiran, could you find some time for a review?

File: Lib/ldap/ldapobject.py [hunk 1]
```diff
@@ -76,7 +76,9 @@ class NO_UNIQUE_ENTRY(ldap.NO_SUCH_OBJECT):
 
 class SimpleLDAPObject:
   """
-  Drop-in wrapper class around _ldap.LDAPObject
+    This basic class wraps all methods of the underlying C API object.
```
Comment (tiran) [inline Lib/ldap/ldapobject.py]: Please also indent the leading and trailing ``"""``.

File: Lib/ldap/ldapobject.py [hunk 2]
```diff
@@ -1057,15 +1059,20 @@ def get_naming_contexts(self):
...
-  In case of server failure (ldap.SERVER_DOWN) the implementations
-  of all synchronous operation methods (search_s() etc.) are doing
-  an automatic reconnect and rebind and will retry the very same
-  operation.
-
-  This is very handy for broken LDAP server implementations
-  (e.g. in Lotus Domino) which drop connections very often making
-  it impossible to have a long-lasting control flow in the
-  application.
+    :py:class:`SimpleLDAPObject` subclass whose synchronous request methods
```
Comment (tiran) [inline Lib/ldap/ldapobject.py]: Please also indent the leading and trailing ``"""``.

File: Lib/ldap/ldapobject.py (hunk 1 above)
Comment (encukou) [reply Lib/ldap/ldapobject.py]: That would mean re-indenting the entire class, which is too much for this PR. So I un-indented the contents instead.

**Pull request merged**